doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -6,7 +6,7 @@
 
 **X-ROAD 6**
 
-Version: 2.36
+Version: 2.38
```

Comment:
Should the use case documentation also be updated, as it contains other API key related operations? https://github.com/nordic-institute/X-Road/blob/735e853ba2b24871b82606d8df77478ba2283ecb/doc/UseCases/uc-ss_x-road_use_case_model_for_security_server_management_1.4_Y-883-4.md#344-uc-ss_43-create-a-new-api-key

Reply:
Yes, it should be updated too. Editing an API key is now added to the use case document.